template-for-repository/proofs/Makefile.common

```diff
@@ -138,6 +138,22 @@ ifneq ($(strip $(EXTERNAL_SAT_SOLVER)),)
 endif
 CHECKFLAGS += $(USE_EXTERNAL_SAT_SOLVER)
 
+# Job pools
+# For version of Litani that are new enough (where `litani print-capabilities`
+# prints "pools"), proofs for which `EXPENSIVE = true` is set can be added to a
+# "job pool" that forces only a single "expensive" proof to run at a time. All
```

Comment:
What does it mean for proofs to run one at a time?  That the property and coverage checking run concurrently but only for one proof at a time, or is everything sequential?   Why isn't totally sequential a massive overkill?  If I have enough memory to run 16 proofs 4 at a time, why should I be happy to run them one at a time?

Reply:
Thanks for this comment! I've expanded the documentation here to answer your questions (see "Improve documentation for job pools feature"), and also made the level of parallelism restriction configurable ("Make parallelism of EXPENSIVE proofs configurable").